src/pages/Cart/Cart.js

```diff
@@ -1,6 +1,92 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import ItemChoice from '../../Components/ItemChoice/ItemChoice';
+import EmptyCart from './EmptyCart/EmptyCart';
+import Item from './Item/Item';
 import './Cart.scss';
 
-const Cart = () => {};
+const Cart = () => {
+  const [productList, setProductList] = useState([]);
+  const [totalPrice, setTotalPrice] = useState(0);
+
+  useEffect(() => {
+    fetch('/data/productList.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setProductList(data);
+      });
+  }, []);
+
+  return (
+    <div className="cart">
+      <h2 className="tit_cart">
+        장바구니 <span className="screen_out">담긴상품 : </span>
+        {productList.length}개
+      </h2>
+      {productList.length === 0 ? (
+        <EmptyCart />
+      ) : (
+        <div className="group_cart">
+          <div className="wrap_listcart">
+            <strong className="screen_out">장바구니에 담긴 상품 목록</strong>
+            <span className="item_choice">
+              <input type="checkbox" id="checkBox" className="inp_check" />
+              <label htmlFor="checkBox" className="lab_check">
+                <span className="ico_shop ico_check"></span>
+              </label>
+              <span className="txt_choice">전체선택</span>
+            </span>
+            <ul className="list_cart">
+              {productList.map(
+                ({ id, image, name, count, price, quantity }) => {
+                  return (
+                    <Item
+                      key={id}
+                      id={id}
+                      image={image}
+                      name={name}
+                      checkBoxName={id}
+                      count={count}
+                      price={price}
+                      quantity={quantity}
+                      total={quantity * price}
+                      totalPrice={totalPrice}
+                      handleChange={() => {}}
+                      productList={productList}
+                      setProductList={setProductList}
+                      setTotalPrice={setTotalPrice}
+                    />
+                  );
+                }
+              )}
```

Comment:
```suggestion
              {productList.map(
                (product) => {
                  return (
                    <Item
                      key={id}
                      product={product}
                      totalPrice={totalPrice}
                      handleChange={() => {}}
                      productList={productList}
                      setProductList={setProductList}
                      setTotalPrice={setTotalPrice}
                    />
                  );
                }
              )}
```
이렇게 보내줘야할 prop이 많을 경우 다 풀어서 보내주기 보다는 product와 같이 한 번에 보내준 후 하위 컴포넌트에서 풀어서 사용하는 것이 더 효율적입니다!

Reply:
수정했습니다~! 확인 부탁드립니다 감사합니다 :)